pandas-stubs/io/json/_json.pyi

```diff
@@ -110,10 +115,10 @@ def read_json(
     storage_options: StorageOptions = ...,
 ) -> DataFrame: ...
 
-class JsonReader(abc.Iterator):
+class JsonReader(abc.Iterator, Generic[FrameSeriesStrT]):
     def read(self) -> DataFrame | Series: ...
     def close(self) -> None: ...
-    def __iter__(self) -> JsonReader: ...
+    def __iter__(self) -> JsonReader[FrameSeriesStrT]: ...
     def __next__(self) -> DataFrame | Series: ...
```

Comment:
Can't this return `FrameSeriesStrT` as well?

Reply:
No longer relevant.